canonical-web-and-design/charmhub.io
# WIP: Test typescript migration

## Done
- _Migrate the project to using both javascript and typescript._

## How to QA
- Run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
- See it works with both `*.js` and `*.ts` files

## Issue / Card
Fixes #

## Screenshots
[if relevant, include a screenshot]
